app/Http/Controllers/Twill/ModuleController.php

```diff
@@ -6,6 +6,11 @@
 
 class ModuleController extends BaseModuleController
 {
+    protected function setPreviewView(string $previewView): void
+    {
+        $this->previewView = $previewView;
+    }
```

Comment:
Surprisingly, there wasn't already a setter for this property, but we use it quite a bit, so I thought I'd create one.